scripts/register_new_matrix_user

```diff
@@ -197,4 +202,10 @@ if __name__ == "__main__":
     else:
         secret = args.shared_secret
 
-    register_new_user(args.user, args.password, args.server_url, secret, args.admin)
+    if args.admin and args.regular_user:
+        print "Both --admin and --regular-user are set, choose either."
+        sys.exit(1)
+
+    admin = True if args.admin else False if args.regular_user else None
```

Comment:
I honestly can't remember how multiple if statements like this work, can we write split it out into actual if statements please?

(You can also write this as `args.admin or args.regular_user or None`, but that's a bit special too)